specification/dartLangSpec.tex

```diff
@@ -9168,33 +9169,52 @@ \subsection{Symbols}
 representing the reserved word \VOID.
 
 \LMHash{}%
-Assume that the term $t$ is an identifier \id{}
-that does not start with an underscore,
-that $t$ is \VOID,
-that $t$ is a period separated sequence of identifiers
-\code{$id_1$.$id_2$.$\cdots$.$id_n$},
-or that $t$ is derived from \synt{operator}.
-Any two occurrences of \code{\#t} evaluate to the same object
-(\commentary{that is, symbol instances are canonicalized}),
-and no other symbol literals evaluate to that object,
-nor to a \code{Symbol} instance that is equal to that object
-according to the \lit{==} operator
+For the value $o$ of a symbol literal representing a source code term
+like an identifier or an operator, we say that $o$ is a
+\IndexCustom{non-private symbol based on}{symbol!non-private, based on}
+the string whose contents is the characters of that term, without whitespace.
```

Comment:
Unless it's a private identifier?

Reply:
The intention is that 'representing' (or 'represents', etc.) is a reference to the 4 occurrences of that word in the previous ~20 lines of text. However, I can see that it is ambiguous, so we should adjust it.

Cf. https://github.com/dart-lang/language/issues/1783.